openedx/core/djangoapps/notifications/tests/test_tasks.py

```diff
@@ -254,6 +258,7 @@ def test_send_with_app_disabled_notifications(self, app_name, notification_type)
         self.assertIsNone(notification)
 
     @override_waffle_flag(ENABLE_NOTIFICATIONS, active=True)
+    @override_waffle_flag(ENABLE_PUSH_NOTIFICATIONS, active=True)
```

Comment:
For test cases, which don't require push notifications to be enabled, I think we should skip `override_waffle_flag` for them